docs/README.md

```diff
@@ -77,15 +77,17 @@ The Benefits application doesn't collect or store any user data directly, and we
 
 Sensitive user information exists in the following places:
 
-- To enroll in a senior discount, users need to [provide personal information to Login.gov](https://benefits.calitp.org/help#login-gov-verify).
-- Users need to [provide their credit or debit card information to our transit processor (Littlepay)](https://benefits.calitp.org/help#littlepay) to enroll in a discount.
+- To enroll for reduced fares as an older adult, U.S. Veteran, or Calfresh cardholder, riders need to [provide personal information to Login.gov](https://benefits.calitp.org/help#login-gov-verify).
+- To enroll for reduced fares as a Medicare cardholder, riders need to [provide personal information to Medicare.gov](https://www.medicare.gov/basics/get-started-with-medicare).
+- Eligible riders need to provide their contactless credit or debit card information to one of our transit processors, [Littlepay](https://benefits.calitp.org/help#littlepay) or [Switchio](https://benefits.calitp.org/help#littlepay), to register their card for reduced fares.
```

Comment:
The links are fine for now. Switchio link to be updated in #2909

Reply:
```suggestion
- Eligible riders need to provide their contactless credit or debit card information to one of our transit processors, [Littlepay or Switchio](https://benefits.calitp.org/help#payment_partners), to register their card for reduced fares.
```

Links directly to https://test-benefits.calitp.org/help#payment_partners

<img width="953" height="280" alt="image" src="https://github.com/user-attachments/assets/a3d3e938-9b17-4d9e-9685-8d23f0c5fece" />